### PR TITLE
Script for symbol search bug

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a simple script to investigate the symbol search bug from v4.21.

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: scripts/check_gene_symbols_changed_in_panels.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+            cli_config = yaml.load(in_handle, Loader=yaml.SafeLoader)
+
+    mongo_config["mongodb"] = mongodb or cli_config.get("mongodb") or "scout"
+
+    mongo_config["host"] = host or cli_config.get("host") or "localhost"
+    mongo_config["port"] = port or cli_config.get("port") or 27017
+    mongo_config["username"] = username or cli_config.get("username")
+    mongo_config["password"] = password or cli_config.get("password")
+    mongo_config["authdb"] = authdb or cli_config.get("authdb") or mongo_config["mongodb"]
+    mongo_config["omim_api_key"] = cli_config.get("omim_api_key")
```
Comment (northwestwitch) [inline scripts/check_gene_symbols_changed_in_panels.py:72]: I'm sure you tested this. Perhaps this param is not required?

**Review by northwestwitch — APPROVED** [review]
👍🏻

File: scripts/check_gene_symbols_changed_in_panels.py (hunk 1 above)
Comment (dnil) [reply scripts/check_gene_symbols_changed_in_panels.py:72]: True enough - not needed. Its just copy paste legacy!

**Comment by northwestwitch** [comment]
You could perhaps post a screenshot with the results of the scripts?

**Comment by dnil** [comment]
Good point! Here goes:

`python check_gene_symbols_changed_in_panels.py --port 27019 --mongodb scout`

Screenshot 2021-05-17 at 10 21 54

Screenshot 2021-05-17 at 10 23 15
Screenshot 2021-05-17 at 10 25 27
Screenshot 2021-05-17 at 10 25 48
Screenshot 2021-05-17 at 10 26 05
Screenshot 2021-05-17 at 10 26 20
Screenshot 2021-05-17 at 10 26 28
Screenshot 2021-05-17 at 10 26 38
Screenshot 2021-05-17 at 10 26 43
Screenshot 2021-05-17 at 10 26 49
Screenshot 2021-05-17 at 10 27 04
Screenshot 2021-05-17 at 10 27 09
Screenshot 2021-05-17 at 10 27 19
Screenshot 2021-05-17 at 10 27 28
Screenshot 2021-05-17 at 10 27 32
Screenshot 2021-05-17 at 10 27 37
Screenshot 2021-05-17 at 10 27 44

**Comment by dnil** [comment]
Its really slow with case-by-case queries compared to the gene_variants. And maybe the mongo instance is getting a little bothered as well by now. Oh well, with no timeout it ought to finish within like the next hour at least.

**Pull request merged**